.drone.yml

```diff
@@ -1,193 +1,342 @@
 kind: pipeline
-name: deb_packaging
+name: Debian 8
+type: exec
 
 platform:
   os: linux
   arch: amd64
 
+node:
+  distro: debian8
+
+# I would like to put a single "environmet" section here and do not copy it
+# to each step with upload. But, unfortunately, "environmet per pipeline" has been supported
+# just for docker runners now. https://docs.drone.io/pipeline/environment/syntax/
+
 steps:
 - name: build Debian Jessie packages
-  image: debian:jessie
-
-  commands:
-    # Step 1: Prepare container
-    - apt-get update &&
-        apt-get install -y build-essential rsync git pax apt-utils tree
-        gnupg2 dpkg-sig
-    - dir=`pwd` && cd / &&
-        git clone https://github.com/debbuild/debbuild.git &&
-        cd /debbuild &&
-        ./configure && make && make install &&
-        cd $dir
-    # Step 2: Build deb packages
+  commands:
+    # Step 1: Build deb packages
     - make deb
-    # Step 3: Collect artifacts
+    # Step 2: Collect artifacts
     - repobuild/collect_artifacts.sh
 
+- name: Upload artifacts
+  environment:
+    AWS_ACCESS_KEY_ID:
+      from_secret: AWS_ACCESS_KEY_ID
+    AWS_SECRET_ACCESS_KEY:
+      from_secret: AWS_SECRET_ACCESS_KEY
+    AWS_DEFAULT_REGION:
+      from_secret: AWS_REGION
+    DEBIAN_FRONTEND: noninteractive
+  commands:
+    - repobuild/upload.sh
+      --source repobuild/artifacts/
+      --backet artifacts.assur.io
```

Comment:
Are you sure that's not a mistake?  The correct spelling is "bucket"

Reply:
Changed here and in the packaging.